lib/common/client/base.rb

```diff
@@ -45,7 +45,16 @@ def perform(method, path, params, headers = nil)
         send(method, path, params || {}, headers || {})
       end
 
+      # rubocop:disable Metrics/MethodLength
       def request(method, path, params = {}, headers = {})
+        log_message_to_sentry(
+          'nil headers bug',
+          :info,
+          headers: headers, method: method, path: path, params: params, client: inspect,
```

Comment:
I'd check that Sentry obfuscates all PII, EVSS and other systems send it along in headers

Reply:
Good call.  Completed in https://github.com/department-of-veterans-affairs/vets-api/pull/1894/commits/e7dc63ee37b7af8128af5b35bd28083276abb160